docs/apache-airflow/core-concepts/dag-run.rst

```diff
@@ -44,42 +44,6 @@ There are two possible terminal states for the DAG Run:
 
 DAGs that have a currently running DAG run can be shown on the UI dashboard in the "Running" tab. Similarly, DAGs whose latest DAG run is marked as failed can be found on the "Failed" tab.
 
-Cron Presets
```

Comment:
I think we need entry in `redirects.txt` so it would be easy to find this information between older and newer versions of Airflow

https://github.com/apache/airflow/blob/475818542e688a2ae2b5d8dce55e6c6f1debdf76/docs/apache-airflow/redirects.txt